docs/generate-features.md

```diff
@@ -0,0 +1,46 @@
+#### generate-features
+---
+Scan the class files of your application and create a file containing the Liberty features that it requires.
```

Comment:
Consider "create a new Liberty configuration file containing" or "create a new file in your Liberty configuration directory"

Reply:
👍